platform_versioned_docs/version-24.2/data_studios/index.mdx

```diff
@@ -26,6 +26,10 @@ Before you get started, you need the following:
 Currently, Data Studios only supports AWS Batch compute environments that **do not** have Fargate enabled.
 :::
 
+## Limitations
+
+If you configured your AWS Batch compute environment to include an EFS file system with **EFS file system > EFS mount path**, the mount path cannot be the same as your work directory. When the work directory and EFS mount path are identical, the Data Studio snapshots fail causing a data studio instance to fail. For more information on AWS Batch configuration, see [AWS Batch][aws-batch].
```

Comment:
Looks good to me!
The only thing I would also add that the mount path cannot be empty either (because then it defaults to the work directory, leading to failure)

```
Enter the EFS mount path (defaults to the pipeline work directory root path if omitted). 
```